app/general/admin.py

```diff
@@ -64,6 +71,16 @@ def clean(self):
         return cleaned_data
 
 
+class DocumentFormWithFulltext(DocumentForm):
+    class Meta:
+        model = Document
+        fields = "__all__"  # noqa: DJ007
+        labels = {"document_data": _("Document full-text (used in search)")}
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
Unneeded?

Reply:
Oops, I used to have something here and deleted it. Good catch!